README.md

```diff
@@ -354,20 +344,108 @@ The CLI accepts a series of files or globs to be formatted, as well as a set of
+| ---------- | --------- | ---------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ---------------------------- |
+| `config`   | `c`       | FORMAT_PACKAGE_CONFIG  | Path to a custom configuration to use. This configuration can be JavaScript, `JSON`, or any other format that your configuration of node can `require`. The default configuration can be found [here](lib/defaults/index.js). |                              |
+| `write`    | `w`       | FORMAT_PACKAGE_WRITE   | Write the output to the location of the found `package.json`                                                                                                                                                                  | **false**                    |
+| `ignore`   | `i`       | FORMAT_PACKAGE_IGNORE  | Patterns for ignoring matching files                                                                                                                                                                                          | **`['**/node_modules/**']`** |
+| `verbose`  | `v`       | FORMAT_PACKAGE_VERBOSE | Print the output of the formatting                                                                                                                                                                                            | **false**                    |
+| `help`     | `h`       |                        | Print help menu                                                                                                                                                                                                               |                              |
 
 You can also see the available options in the terminal by running:
 
 ```
 yarn format-package --help
 ```
 
+## Config
+
+`format-package` supports a custom configuration from:
+
+- a path specified with `--config`
+- a config file relative to `process.cwd()`
+- a path from an environment variable `FORMAT_PACKAGE_CONFIG`
+
+### with `--config`
+
+When using `--config` for custom configuration, this can be a module name
+or a file relative to `process.cwd()`.
+
+```
+yarn format-package -c @acme/format-config package.json
+yarn format-package -c ./configs/format-config.js package.json
+yarn format-package -c format-config.js package.json
+```
+
+### with a config file
+
+```
+- format-package property in package.json
+- format-package.json, format-package.yaml or format-package.yml
+- format-package.config.js file exporting a JS object
+- format-package.config.yaml or format-package.config.yml file in YAML format
+```
+
+### with an environment variable `FORMAT_PACKAGE_CONFIG`
+
```

Comment:
A small bit of text (1 sentence) here explaining what is happening would be nice for any developers who are not be super familiar with command line tools and could be unaware of how an environment variable works.